pkg/apis/pingcap/v1alpha1/types.go

```diff
@@ -2020,12 +2065,18 @@ type RestoreSpec struct {
 	To *TiDBAccessConfig `json:"to,omitempty"`
 	// Type is the backup type for tidb cluster.
 	Type BackupType `json:"backupType,omitempty"`
+	// RestoreMode is the restore mode. such as snapshot or pitr.
+	RestoreMode RestoreMode `json:"restoreMode,omitempty"`
+	// RestoreTs is the pitr restore ts.
+	RestoreTs string `json:"restoreTs,omitempty"`
 	// TikvGCLifeTime is to specify the safe gc life time for restore.
 	// The time limit during which data is retained for each GC, in the format of Go Duration.
 	// When a GC happens, the current time minus this value is the safe point.
 	TikvGCLifeTime *string `json:"tikvGCLifeTime,omitempty"`
 	// StorageProvider configures where and how backups should be stored.
 	StorageProvider `json:",inline"`
+	// LogBackupProvider configures where and how log backup should be stored.
+	LogBackupProvider StorageProvider `json:"logBackupProvider,omitempty"`
```

Comment:
LogBackupStorageProvider would be more accurate

Reply:
ok